### PR TITLE
ShwetaD_Assignment12

### PR DESCRIPTION
**Pull request by ShwetaDharmadhikari**
image

### CODE REVIEW

File: src/shweta_Dharmadhikari/Mall.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	String paymentDoneUsing(String paymentMethod) {
+		//payMethod = paymentMethod;
+		return paymentMethod;
+	}
+	
+	void displayCutomerInfo(String name, int item,String payMethod,String sectionName) {
+		System.out.println(name+" brought "+item+" items and payment done using "+payMethod+" in section "+sectionName);
+	}
+	
+	boolean needBill(boolean isBillRequired) {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/Mall.java:22]: If else block not required, return isBillRequired.

File: src/shweta_Dharmadhikari/Mall.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		if (isBillRequired)
+			return true;
+		else
+			return false;
+	}
+	
+	boolean haveCarryBag(boolean isCarryBagRequired) {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/Mall.java:29]: If else block not required, return isCarryBagRequired.

File: src/shweta_Dharmadhikari/Mall.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		if (isCarryBagRequired)
+			return true;
+		else
+			return false;
+	}
+	
+	String decideSection(int item,String payMethod) {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/Mall.java:36]: For item=10, section will be blank.

**Review by Upasana1703 — APPROVED** [review]

File: src/shweta_Dharmadhikari/Mall.java (hunk 2 above)
Comment (ShwetaDharmadhikari) [reply src/shweta_Dharmadhikari/Mall.java:29]: nope , customer will send true or false from main method so we have to set isCarryBagRequired variable

**Pull request merged**